.github/workflows/python.yml

```diff
@@ -0,0 +1,115 @@
+name: Python bindings
+on: [push, pull_request]
+
+env:
+  CIBW_ENVIRONMENT_LINUX: CCACHE_DIR=/host/home/runner/.ccache
+  CIBW_ENVIRONMENT_WINDOWS: PATH="$PATH;C:\\Boost\\bin" LIB="$LIB;C:\\Boost\\lib" INCLUDE="C:\\Boost\\include" VSCMD_ARG_TGT_ARCH=""
+  CIBW_ENVIRONMENT_MACOS: MACOSX_DEPLOYMENT_TARGET=10.14
+
+  CIBW_BEFORE_ALL: bash {package}/bindings/python/tools/install_compiler_cache.sh
+  CIBW_BEFORE_BUILD: bash {package}/bindings/python/tools/install_boost.sh {package}
+  CIBW_TEST_COMMAND: python {package}/bindings/python/test/test.py -b
+
+  CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
+  CIBW_MANYLINUX_I686_IMAGE: manylinux2014
+
+  CIBW_SKIP: cp27-* pp*
+
+jobs:
+  build:
+    name: Building and testing on ${{ matrix.config.os }} ${{ matrix.config.architecture }}
+    runs-on: ${{ matrix.config.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+        - {os: ubuntu-latest, architecture: "*x86_64" }
+        - {os: ubuntu-latest, architecture: "*i686" }
+        - {os: windows-latest, architecture: "*amd64" }
+        - {os: windows-latest, architecture: "*win32" }
+        - {os: macos-latest, architecture: "*x86_64" }
```

Comment:
I don't see how "architecture" is used anywhere it matters. Just to name the ccache on linux

Reply:
I suppose it sets `CIBW_BUILD`, which it looks like is used by `cibuildwheel`

Reply:
It is used [here](https://github.com/arvidn/libtorrent/pull/4218/files#diff-092659a9bd0068791326fb1d08f005532e5aeb983b999a7bd32c51deee0a71d5R34) to set wheel architecture (using [`CIBW_BUILD`](https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip) env var) for cibuildwheel. This is used to make builds for different architectures parallel and faster.